src/main/kotlin/platform/mixin/inspection/StaticMemberInspection.kt

```diff
@@ -33,6 +34,7 @@ import com.intellij.psi.PsiField
 import com.intellij.psi.PsiMember
 import com.intellij.psi.PsiMethod
 import com.intellij.psi.PsiModifier
+import org.jetbrains.kotlin.j2k.accessModifier
```

Comment:
This will break when the Kotlin plugin is not installed. Please find an alternative API.